packages/main/src/SegmentedButton.ts

```diff
@@ -289,12 +325,26 @@ class SegmentedButton extends UI5Element {
 	 * @readonly
 	 * @type {sap.ui.webc.main.ISegmentedButtonItem}
 	 * @name sap.ui.webc.main.SegmentedButton.prototype.selectedItem
+	 * @deprecated since 1.14.0. This method will be removed in the next major release.
+	 * Please use the <code>selectedItems</code> property instead.
 	 * @public
 	 */
 	get selectedItem() {
 		return this._selectedItem;
 	}
 
+	/**
+	 * Returns an array of the currently pressed items.
```

Comment:
Maybe pressed -> selected in the jsdoc as well

Reply:
done